html-web-page/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML Web Page</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      td:not(:first-child) {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <!-- your HTML code here 👇 -->
+
+    <h1>Jelly of the Month Club</h1>
```

Comment:
I would put this line inside of a `header`. `header` is a semantic element so it will help with accessibility & SEO.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header